extensions/amp-analytics/0.1/events.js

```diff
@@ -667,9 +682,36 @@ class TimerEventHandler {
     }
     win.clearInterval(this.intervalId_);
     this.intervalId_ = undefined;
+    this.calculateDuration_();
+    this.lastPingTime_ = undefined;
+    this.intervalCallback_();
```

Comment:
can this be moved to `stopTimer_` function? 
normally i'd not expect to have the callback called in a clear function.

Reply:
Renamed for clarity.